### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by wongJonathan**
used for travis ci

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ describe('Given invalid input for ', () => {
 });
 
 describe('Catching errors in updateVpcConfig ', () => {
-  it('AWS api call describeVpcs fails', () => {
+  it('AWS api call describeVpcs fails', (done) => {
```
Comment (aoskotsky-amplify) [inline test/index.test.js]: You shouldn't need to do this. Returning the promise should be sufficient. Try checking that travis is running the right version of node/mocha

Comment (wongJonathan) [inline test/index.test.js]: It should be downloading the latest, it seems to get held up because of a timeout error.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+script:
+  - npm run lint
+  - bash test/versionCheck.sh $TRAVIS_BRANCH $TRAVIS_PULL_REQUEST
+  - npm test
+
+deploy:
+  provider: npm
+  api_key: "API KEY HERE"
+  on:
+    branch: master
```
Comment (aoskotsky-amplify) [inline .travis.yml]: new line

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - npm install
+
+script:
+  - npm run lint
+  - bash test/versionCheck.sh $TRAVIS_BRANCH $TRAVIS_PULL_REQUEST
+  - npm test
+
+deploy:
+  provider: npm
+  api_key: "API KEY HERE"
```
Comment (aoskotsky-amplify) [inline .travis.yml]: You should be able to put the key here using travis secrets

Comment (wongJonathan) [inline .travis.yml]: I don't have the key yet so I just put it there as a place holder.

**Pull request merged**